src/MigrationHelper.php

```diff
@@ -13,59 +13,32 @@
 use Yiisoft\Db\Schema\SchemaBuilderTrait;
 use Yiisoft\Strings\StringHelper;
 
-use function implode;
-
-/**
- * Migration is the base class for representing a database migration.
- *
- * Migration is designed to be used together with the "vendor/bin/yii migrate" command.
- *
- * Each child class of Migration represents an individual database migration which is identified by the child class
- * name.
- *
- * Within each migration, the {@see up() method should be overridden to contain the logic for "upgrading" the database;
- * while the {@see down()} method for the "downgrading" logic. The "yii migrate" command manages all available
- * migrations in an application.
- *
- * If the database supports transactions, if anything wrong happens during the upgrading or downgrading, the whole
- * migration can be reverted in a whole.
- *
- * Note that some DB queries in some DBMS cannot be put into a transaction. For some examples, please refer to
- * [implicit commit](http://dev.mysql.com/doc/refman/5.7/en/implicit-commit.html). If this is the case, you should still
- * implement `up()` and `down()`, instead.
- *
- * Migration provides a set of convenient methods for manipulating database data and schema.
- *
- * For example, the {@see insert()} method can be used to easily insert a row of data into a database table; the
- * {@see createTable()} method can be used to create a database table.
- *
- * Compared with the same methods in {@see Command}, these methods will display extra information showing the method
- * parameters and execution time, which may be useful when applying migrations.
- *
- * For more details and usage information on Migration, see the [guide article on Migration](guide:db-migrations).
- */
-abstract class Migration implements MigrationInterface
+final class MigrationHelper
```

Comment:
May not be the best name. Helpers are usually static classes.

Reply:
`DbMigrator $db`
`Migrator $m`
?

Reply:
Migrator sounds like something that applies migrations.

Reply:
`Db`
`MigrationQuery`
?

Reply:
`MigrationBuilder`?

Reply:
`MigrationBuilder` soulds alright. @vjik let's apply it.

Reply:
> `MigrationBuilder` soulds alright. @vjik let's apply it.

Which variable user in migrations?
I like `$b`, but can `$mb`...

Reply:
`$b` is OK.

Reply:
Done.